content/_data/upgrades/2-235-1.adoc

```diff
@@ -0,0 +1,18 @@
+==== Plugin manager improvements
+
+The plugin manager interface includes significant improvements.
+Those improvements include a change in the "Available" tab.
+
+The "Available" tab previously listed all available plugins then filtered based on the value in the "search" field.
+The "Available" tab now lists no plugins until the user enters a value in the "search" field.
+Users who previously scrolled through over 1000 plugins to find their plugin will now need to enter a value in the search field.
+
+==== Auto refresh removed
```

Comment:
Add a Jira reference?

Reply:
Thanks.  Done in 9c9751444a46c29afb256c55c5db0f66598fde80